miden/src/cli/prove.rs

```diff
@@ -44,6 +44,10 @@ pub struct ProveCmd {
     #[clap(short = 'r', long = "recursive")]
     recursive: bool,
 
+    /// Specifies if the RPX Hash should be used. It requires the recursive flag to be unset
+    #[clap(long = "rpx")]
+    rpx: bool,
```

Comment:
question: is there a way to specify that this should not be used together with `--recursive` flag?

Reply:
Yes, i found a way by using `conflicts_with` attribute. Tested the cli by calling them both together and it outputs correct error message about it.